modules/simple-payments/simple-payments.php

```diff
@@ -286,6 +287,134 @@ function allow_sync_post_meta( $post_meta ) {
 		) );
 	}
 
+	/**
+	 * Enable Simple payments custom meta values for access through the REST API.
+	 * Field’s value will be exposed on a .meta key in the endpoint response,
+	 * and WordPress will handle setting up the callbacks for reading and writing
+	 * to that meta key.
+	 *
+	 * @link https://developer.wordpress.org/rest-api/extending-the-rest-api/modifying-responses/
+	 */
+	public function register_meta_fields_in_rest_api() {
+		register_meta( 'post', 'spay_price', array(
+			'description'       => esc_html__( 'Simple payments; price.', 'jetpack' ),
+			'object_subtype'    => self::$post_type_product,
+			'sanitize_callback' => array( $this, 'sanitize_price' ),
+			'show_in_rest'      => true,
+			'single'            => true,
+			'type'              => 'number',
+		) );
+
+		register_meta( 'post', 'spay_currency', array(
+			'description'       => esc_html__( 'Simple payments; currency code.', 'jetpack' ),
+			'object_subtype'    => self::$post_type_product,
+			'sanitize_callback' => array( $this, 'sanitize_currency' ),
+			'show_in_rest'      => true,
+			'single'            => true,
+			'type'              => 'string',
+		) );
+
+		register_meta( 'post', 'spay_cta', array(
+			'description'       => esc_html__( 'Simple payments; text with "Buy" or other CTA', 'jetpack' ),
+			'object_subtype'    => self::$post_type_product,
+			'sanitize_callback' => 'sanitize_text_field',
+			'show_in_rest'      => true,
+			'single'            => true,
+			'type'              => 'string',
+		) );
+
+		register_meta( 'post', 'spay_multiple', array(
+			'description'       => esc_html__( 'Simple payments; allow multiple items', 'jetpack' ),
+			'object_subtype'    => self::$post_type_product,
+			'sanitize_callback' => 'absint',
+			'show_in_rest'      => true,
+			'single'            => true,
+			'type'              => 'integer',
+		) );
+
+		register_meta( 'post', 'spay_email', array(
+			'description'       => esc_html__( 'Simple payments button; paypal email.', 'jetpack' ),
+			'sanitize_callback' => 'sanitize_email',
+			'show_in_rest'      => true,
+			'single'            => true,
+			'type'              => 'string',
+		) );
+
+		register_meta( 'post', 'spay_formatted_price', array(
```

Comment:
As I've mentioned on https://github.com/Automattic/wp-calypso/pull/28169#discussion_r230430737 we are safe to remove this `meta`. I'm working on removing it permanently so there's no more confusion with `spay_price`.

Reply:
👍 Let's drop this, I wasn't involved in the conversation but having the client push a formatted price raised some red flags for me when I was initially looking at this.

Reply:
Removed in https://github.com/Automattic/jetpack/pull/10402/commits/3e393a3c57f0c810ac218a4cb322dcc123265f2e

Reply:
❤️